src/rpc/blockchain.cpp

```diff
@@ -1867,7 +1907,13 @@ static UniValue getblockstats(const JSONRPCRequest& request)
     ret_all.pushKV("maxfeerate", maxfeerate);
     ret_all.pushKV("maxtxsize", maxtxsize);
     ret_all.pushKV("medianfee", CalculateTruncatedMedian(fee_array));
-    ret_all.pushKV("medianfeerate", CalculateTruncatedMedian(feerate_array));
+
+    UniValue feerates_res(UniValue::VARR);
```

Comment:
nit: I'd push this code up to immediately below the `CalculatePercentilesByWeight()` call so all the `ret_all.pushKV()` calls are on contiguous (alphabetized) lines.